toolkit/resources/assets/grub2/grub.cfg

```diff
@@ -8,11 +8,14 @@ if [ -f  $bootprefix/systemd.cfg ]; then
 else
 	set systemd_cmdline=net.ifnames=0
 fi
+if [ -f $bootprefix/grub2/grubenv ]; then
+	load_env -f $bootprefix/grub2/grubenv
+fi
 
 set rootdevice={{.RootPartition}}
 
 menuentry "CBL-Mariner" {
-	linux $bootprefix/$mariner_linux {{.LuksUUID}} {{.LVM}} {{.IMAPolicy}} {{.ReadOnlyVerityRoot}} {{.SELinux}} rd.auto=1 root=$rootdevice $mariner_cmdline lockdown=integrity sysctl.kernel.unprivileged_bpf_disabled=1 $systemd_cmdline {{.ExtraCommandLine}}
+	linux $bootprefix/$mariner_linux {{.LuksUUID}} {{.LVM}} {{.IMAPolicy}} {{.ReadOnlyVerityRoot}} {{.SELinux}} rd.auto=1 root=$rootdevice $mariner_cmdline lockdown=integrity sysctl.kernel.unprivileged_bpf_disabled=1 $systemd_cmdline $kernelopts {{.ExtraCommandLine}}
```

Comment:
I'm wondering if the grubenv's $kernelopts should be last, that way editing the grubenv would override the entries passed at image build time (ExtraCommandLine).

Reply:
oh interesting, would you recommend moving it to the front so it has the lowest priority?

Reply:
It should probably have the highest priority, since this is effectively trying to overwrite the existing configs